ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/retentionpolicy/creation/PointInTimeRetentionPolicyCreator.java

```diff
@@ -0,0 +1,12 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.creation;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.RetentionPolicy;
+import org.apache.jena.rdf.model.Model;
+
+public class PointInTimeRetentionPolicyCreator implements RetentionPolicyCreator {
+	@Override
+	public RetentionPolicy createRetentionPolicy(Model model) {
+		// TODO VSDSPUB-718
+		return null;
+	}
+}
```

Comment:
Like already brought up in our dev sessions, our PRs should be focused on working code. This seems like something that should be added in the 718 story. until then, this code will have no use?

Reply:
Apologies!